packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 1.11.0 - (September 27, 2022)
+
 * Added
   * updating docs for disclosure-manager and modal-manager for modal-sizing explainations
```

Comment:
```suggestion
  * updating docs for disclosure-manager and modal-manager for modal-sizing explanations
```

Reply:
corrected in https://github.com/cerner/terra-framework/pull/1575/commits/3e22f32bc10b23da3081c4747ff7f512a14921e9